src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -18,450 +18,700 @@ should be included in the pipeline documentation.
+| GradientNonLinearityGeometryCorrection | OPTIONAL. Boolean. Correction for geometric distortions arising from non-linearity of gradients                                                   |
+| GradientNonLinearityQSpaceCorrection   | OPTIONAL. Boolean. Correction for spatial inhomogeneity of diffusion sensitisation gradient strength                                              |
+| SliceDropoutDetection                  | OPTIONAL. Boolean. Detection of signal dropout in acquired slices during pre-processing                                                           |
+| SliceDropoutReplacement                | OPTIONAL. Boolean. Replacement of image data within slices containing signal dropout with predicted values                                        |
+| BiasFieldCorrectionMethod              | OPTIONAL. String. Method (if any) used for correction of B1 RF field inhomogeneity                                                                |
 
-## Diffusion Models (and input parameters)
+## Diffusion models
 
-Diffusion MRI can be modeled using various paradigms to extract a more easily
-understandable representation of the diffusion process and the underlying
-structure. To do so, parameters might be needed to control how the signal is fit
-to the model. Those parameters are called **input parameters** in the following.
-Once the model is fit, the resulting representation can be saved using a number
-of values per voxel. Those values will be called **output** or **estimated
-parameters** in the following.
+Diffusion MRI can be modeled using various paradigms to extract more
+informative representations of the diffusion process and the underlying
+biological structure. A wide range of such models are available, each of
+which has its own unique requirements with respect to:
 
-**Estimated parameters** are saved as NIFTI files (see section Models for the
-expected content of each model), and **input parameters** are saved in the
-sidecar JSON file.
+-  The [input](#paramdef-input) parameters required in order to
+   define / constrain the model;
 
-The following is a general example of naming convention:
+-  The appropriate [data representations](#data-representations) utilised to
+   store information parameterised [by](#paramdef-intrinsic) or
+   [from](#paramdef-extrinsic) the model onto the filesystem;
+
+-  The requirements for encapsulation and complete representation of
+   derived [orientation information](#orientation-specification), which is
+   a key strength of diffusion MRI but presents unique challenges for
+   correct interpretation.
+
+### Parameter terminology
+
+Throughout this document, the term "parameter" is used to refer to
+multiple distinct sources of information. The distinction between
+these uses is defined thus:
+
+1. <a name="paramdef-input">*Input*</a> model parameter:
+
+   Value or non-numerical setting that influences the conformation
+   of the diffusion model to the empirical diffusion-weighted data.

```

Comment:
Could we add a simple example of each of the 3 types of parameters so that first time readers of the spec who have a dMRI knowledge can quickly map them in their mind? For example, saying that the SH order is an input parameter.

Reply:
I think a lot of the pre-processed data fields may end up disappearing entirely due to not wanting to manage provenance within BIDS.

> Could we add a simple example of each of the 3 types of parameters

Yes, I think throwing in an example here could be beneficial.

Reply:
> > Could we add a simple example of each of the 3 types of parameters

> Yes, I think throwing in an example here could be beneficial.

Added in 1c7b67f.